### PR TITLE
Archive (a.k.a. obsolescence)

### PR DESCRIPTION
**Pull request by adRn-s**

### CODE REVIEW

File: parkour_app/common/models.py [hunk 1]
```diff
@@ -3,15 +3,20 @@
 
 
 def get_deleted_org():
-    return Organization.objects.get_or_create(name="deleted ORG")[0]
+    return Organization.objects.filter(archived=False).get_or_create(
```
Comment (adRn-s) [inline parkour_app/common/models.py]: There's no need to apply the filter here.

File: parkour_app/common/models.py [hunk 2]
```diff
@@ -3,15 +3,20 @@
...
+        name="deleted ORG"
+    )[0]
 
 
 def get_deleted_pi():
-    return PrincipalInvestigator.objects.get_or_create(name="deleted PI")[0]
+    return PrincipalInvestigator.objects.filter(archived=False).get_or_create(
```
Comment (adRn-s) [inline parkour_app/common/models.py]: There's no need to apply the filter here.

File: parkour_app/flowcell/fixtures/sequencer.json
Comment (adRn-s) [inline parkour_app/flowcell/fixtures/sequencer.json:1]: Good call!

File: parkour_app/library_preparation/signals.py [hunk 3]
```diff
@@ -27,6 +27,8 @@ def update_samples(sender, instance, action, **kwargs):
 
         # TODO: maybe there is a better way to create multiple objects at once
         for sample in instance.samples.all():
-            obj, created = LibraryPreparation.objects.get_or_create(sample=sample)
```
Comment (adRn-s) [inline parkour_app/library_preparation/signals.py]: Again, I daresay that all the `get_or_create` calls don't need the filter.

File: parkour_app/migrations/tests.py
Comment (adRn-s) [inline parkour_app/migrations/tests.py:1]: Did I just delete this file by accident with the commit to reset migrations? Please see that it's restored.

File: parkour_app/pooling/signals.py [hunk 4]
```diff
@@ -35,7 +35,9 @@ def create_pooling_objects_sample(sender, instance, **kwargs):
         return
 
     try:
-        lib_prep_object = LibraryPreparation.objects.get(sample=instance)
```
Comment (adRn-s) [inline parkour_app/pooling/signals.py]: Why would the filter be required here?

**Review by adRn-s — COMMENTED** [review]
Good job! We can go through the comments 1 by 1 together, maybe some do prompt for some adjustments, others may be just discussed and leave as-is ⚡  👍🏽

File: parkour_app/common/models.py (hunk 2 above)
Comment (hash-bash) [reply parkour_app/common/models.py]: Got it!

File: parkour_app/common/models.py (hunk 1 above)
Comment (hash-bash) [reply parkour_app/common/models.py]: Yes sure I will remove it!

File: parkour_app/library_preparation/signals.py (hunk 3 above)
Comment (hash-bash) [reply parkour_app/library_preparation/signals.py]: Sure sure, I will remove this one too.

File: parkour_app/migrations/tests.py
Comment (hash-bash) [reply parkour_app/migrations/tests.py:1]: Yes, I will recover it.

File: parkour_app/pooling/signals.py (hunk 4 above)
Comment (hash-bash) [reply parkour_app/pooling/signals.py]: I thought to make it consistent wherever the objects of the data are being used but yes, I will remove this.

**Review by adRn-s — COMMENTED** [review]
Cool!

**Comment by adRn-s** [comment]
reviewing the duplicate #62

**Pull request closed without merging**